jaeger-core/src/main/java/com/uber/jaeger/reporters/RemoteReporter.java

```diff
@@ -93,13 +93,13 @@ public void close() {
       boolean added = commandQueue
           .offer(new CloseCommand(), closeEnqueueTimeout, TimeUnit.MILLISECONDS);
       if (added) {
-        queueProcessorThread.join();
+        queueProcessorThread.join(10000);
```

Comment:
This timeout seems a bit low to me, should we bump to 10s?

Reply:
That's actually 10s (10_000 ms, but the source code compatibility with java6 won't allow me to add the underscore)